_data/releases.yml

```diff
@@ -0,0 +1,18 @@
+#Release versions
+
+legacy:
```

Comment:
Is this necessary?  If not I'd leave it out until we have a legacy version to report here.